.idea/misc.xml

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <project version="4">
-  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (gender_analysis_web)" project-jdk-type="Python SDK" />
+  <component name="ProjectRootManager" version="2" project-jdk-name="Python 3.9 (venv) (2)" project-jdk-type="Python SDK" />
```

Comment:
This file and `.idea/temp.xml` are autogenerated configuration files for your IDE (PyCharm). This change doesn't look super intentional or meaningful. Mind copying out the changed value and restoring it to what came before?